liquid_node/configuration.py

```diff
@@ -160,6 +160,9 @@ def __init__(self):
         self.tika_count = self.ini.getint('liquid', 'tika_count', fallback=1)
         self.tika_memory_limit = self.ini.getint('liquid', 'tika_memory_limit', fallback=800)
 
+        self.thumbnail_count = self.ini.getint('liquid', 'thumbnail_count', fallback=1)
```

Comment:
i'd like these in the `[snoop]` category alongside the
```
        self.snoop_pdf_preview_enabled = self.ini.getboolean('snoop', 'pdf_preview_enabled', fallback=False)
```

setting I added below.

Yes, the `tika_count` and friends should also be moved into that category, but that's a breaking change that's not worth it at the moment.